tool/tbot/main.go

```diff
@@ -158,6 +166,18 @@ func Run(args []string, stdout io.Writer) error {
 		"Arguments to `tsh proxy ...`; prefix with `-- ` to ensure flags are passed correctly.",
 	))
 
+	sshCmd := app.Command("ssh", "").Hidden()
+	sshCmd.Flag("destination-dir", "The destination directory with which to authenticate tsh").StringVar(&cf.DestinationDir)
+
+	sshProxyCmd := sshCmd.Command("proxy", "")
```

Comment:
Just to clarify that this is not some listener (like the various `tsh proxy` subcommands) and that this is intended specifically for use as an openssh `ProxyCommand`:

```suggestion
	sshProxyCmd := sshCmd.Command("ssh-proxy-command", "")
```